graphics.h

```diff
@@ -0,0 +1,30 @@
+#ifndef GRAPHICS_H
+#define GRAPHICS_H
+
+#include <SDL2/SDL.h>
+#include <iostream>
+
+
+ //constants
+ constexpr int kWindowWidth = 640;
+ constexpr int kWindowHeight = 480;
+ constexpr char kGameTitle[] = "DBZ";
+
+ //holds RGB values of background
+ constexpr struct {
+  char red = 0x0;
+  char green = 0x0;
+  char blue = 0xFF;
+
+  //opaque SDL constant
+  char alpha = SDL_ALPHA_OPAQUE;
+
+}BackgroundColor;//background colors
+
+class Graphics {
+public:
+ //initialize SDL library
+ bool InitSdl(); 
+ int createWindow(); 
```

Comment:
CreateWindow, not createWindow.

Header files are the documentation of your class. People using the class will check the header to see what the functions do.

All public functions should be described well in the header. Add a comment above each function describing what it does, what inputs it takes, and what it returns.